src/zl-index.ts

```
@@ -106,6 +107,51 @@
+    return;
+  }
+  const [owner, repoName] = repo.split("/");
+  const fetch = (await import('node-fetch')).default;
+  for (const t of tasks) {
+    // Check for existing open issues with the same title
+    const searchUrl = `https://api.github.com/search/issues?q=repo:${owner}/${repoName}+in:title+${encodeURIComponent(t.task)}`;
+    const searchResp = await fetch(searchUrl, {
+      headers: { 'Authorization': `token ${token}`, 'Accept': 'application/vnd.github.v3+json' }
+    });
+    const searchData = await searchResp.json();
+    if (searchData.items && searchData.items.some((i: any) => i.title === t.task && i.state === 'open')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)